tests/Backend/CommonPart1/DeleteRowsTest.php

```diff
@@ -91,13 +87,13 @@ public function tableDeleteRowsByFiltersData(): array
             ],
             'since yesterday - timestamp' => [
                 [
-                    'changedSince' => $yesterday->getTimestamp(),
+                    'changedSince' => (string) $yesterday->getTimestamp(),
```

Comment:
tohle udelat nemuzeme. Test to sice fixuje, ale je to BC break. Prepnuti `apiDeleteParams` -> `apiDeleteParamsJson` zpusobi, ze ten metoda `deleteTableRows` nebude fungovat (bude strilet 400), kdyz tam nekdo posle klientem timestamp jako int. Takze navrhuju, aby to klient v metode `deleteTableRows` checknout a zkonvertovat na string. Jinak to bude BC break.

Nebo to pak vydat jako minor/major a informovat o tom, aby si minimalne sluzby to osetrily. 

Nebo to vyresit v connection v RO

Reply:
hmm, dobrej point. Sel jsem primarne podle dokumentace, tam je klasicky `string`.
Takze to upravim v Connection, aby to prijimalo int|string a bude to osetreny...

Reply:
fixnuto